tests/acceptance/test_mender-connect.py

```diff
@@ -305,11 +322,11 @@ def test_mender_connect_reconnect(
             )
 
             dbus_set_token_and_url(connection, "", "")
-            kill_time = qemu_system_time(connection)
             # kill the server and wait for error
             with_mock_servers[1].kill()
+            kill_time = qemu_system_time(connection)
             _ = wait_for_string_in_log(
-                connection, kill_time, 300, "error reconnecting:",
+                connection, kill_time, 300, "waiting for reconnect",
```

Comment:
Why won't in this case eventually come back to `error reconnecting`? What has changed in `mender-connect` for this case? :thinking:

Reply:
Based on returns made in the the tests it returned **waiting for reconnect** given the test case, while the test was made to wait for **error reconnecting** which made it fail.

Reply:
Okey